### ISSUE TITLE
Will this platform allows me to show question paper in PDF itself?

### ISSUE THREAD

**Issue by abhisheksahnii** [tecnickcom/tcexam#436]
Will this platform allow me to show/display the question paper in PDF format at left side and allows users just to select the objectives on the right side of the computer screen.

Also, does it support safe exam browser?

**Comment by abhisheksahnii** [comment]
![Screenshot](https://github.com/user-attachments/assets/7a0b1a0d-42b6-443b-b592-eac5175bcca8)


Scenario I am talking about.

**Comment by nicolaasuni** [comment]
Yes. TCExam generates a printable question sheet plus an OMR answer sheet as PDF for pen-and-paper testing — in the admin Test editor use the "generate offline test" action; scanned OMR sheets can then be imported for automatic scoring. You can also export the question bank to PDF. Closing as answered.
